cl/simple_pages/templates/help/prayer_help.html

```diff
@@ -102,8 +102,14 @@ <h3 id="creating-prayer">Creating a Prayer</h3>
   <h3 id="limitations">Limitations</h3>
   <p>Each user can place <span class="bold">{{ daily_quota }}</span> prayers per day. This prevents people from praying for every item on the site.
   </p>
+  <p>Members can place 15 prayers per day.
```

Comment:
Let's use the `daily_quota` variable instead of hardcoding the number. Additionally, I suggest we apply a consistent highlighting style to the maximum prayer count for members
```suggestion
  <p>Members can place <span class="bold">{% widthratio daily_quota 1 3 %}</span> prayers per day.
```